.gitignore

```diff
@@ -0,0 +1 @@
+venv
```

Comment:
I looked at a few open source python projects and the virtual environments have been in the gitignore, so I agree that this is the way to go. Nice catch!